src/main/scala/com/typesafe/sbt/SbtProguard.scala

```diff
@@ -26,19 +32,30 @@ object SbtProguard extends AutoPlugin {
     proguardVersion := "7.0.0",
     proguardDirectory := crossTarget.value / "proguard",
     proguardConfiguration := proguardDirectory.value / "configuration.pro",
-    artifactPath := proguardDirectory.value / (artifactPath in packageBin in Compile).value.getName,
+    artifactPath := proguardDirectory.value / ( Compile / packageBin / artifactPath).value.getName,
     managedClasspath := Classpaths.managedJars(configuration.value, classpathTypes.value, update.value),
-    proguardBinaryDeps := getAllBinaryDeps.value,
-    proguardInputs := (fullClasspath in Runtime).value.files,
-    proguardLibraries := proguardBinaryDeps.value filterNot proguardInputs.value.toSet,
+    proguardBinaryDeps := {
+      (Compile / libraryDependencies).value match {
+        case analysis: Analysis =>
+          analysis.relations.allLibraryDeps.collect { case vf: PathBasedFile =>
+            vf.toPath.toFile
+          }.toSeq
+      }
+    },
```

Comment:
Looks like this is no longer used.

Previously, the binary deps were pulled from the compile analysis to include all binaries, including Java's `rt.jar`. Looks like that's no longer included under library deps, and I see there's a java home added instead.

Would be good to make sure there's still something that works here across Java versions and environments.

Reply:
Yeah, using this method the Java libraries were missing, which is why I added an explicit dependency on `JAVA_HOME`